html-forms/html-forms/index.html

```diff
@@ -0,0 +1,49 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <title>html-forms</title>
+  </head>
+  <body>
+    <h1><b>Coding Form!</b></h1>
+    <form>
+      <div>
+        <label>Email<br  /><input type="text" name="email"></label>
```

Comment:
make sure to avoid using `br` tags for the most part in HTML. If you need an element to wrap onto a new line, a `div` is the way to go!